tests/acceptance/pageObjects/FilesPageElement/filesList.js

```diff
@@ -419,18 +419,25 @@ module.exports = {
       return filesCount + foldersCount
     },
 
-    getShareIndicatorsForResource: async function(fileName) {
+    getShareIndicatorsForResource: async function(fileName, sharingIndicatorVisibility) {
```

Comment:
I feel like that new variable name needs to be clearer, also please add some docs
```suggestion
    getShareIndicatorsForResource: async function(fileName, sharingIndicatorExpectedToBeVisibility) {
```